db/system_keyspace.cc

```diff
@@ -85,9 +85,9 @@ api::timestamp_type system_keyspace::schema_creation_timestamp() {
 // FIXME: Make automatic by calculating from schema structure.
 static const uint16_t version_sequence_number = 1;
 
-table_schema_version system_keyspace::generate_schema_version(utils::UUID table_id, uint16_t offset) {
+table_schema_version system_keyspace::generate_schema_version(::table_id table_id, uint16_t offset) {
     md5_hasher h;
-    feed_hash(h, table_id);
+    feed_hash(h, table_id.uuid());
```

Comment:
We should have feed_hash(h, that_template) so this isn't needed.

Reply:
Yes.  This does the trick:
```
diff --git a/utils/UUID.hh b/utils/UUID.hh
index 1d86249135..8635c07a5f 100644
--- a/utils/UUID.hh
+++ b/utils/UUID.hh
@@ -254,6 +254,14 @@ struct appending_hash<utils::UUID> {
     }
 };
 
+template<typename Tag>
+struct appending_hash<utils::tagged_uuid<Tag>> {
+    template<typename Hasher>
+    void operator()(Hasher& h, const utils::tagged_uuid<Tag>& id) const noexcept {
+        appending_hash<utils::UUID>{}(h, id.uuid());
+    }
+};
+
 namespace std {
 template<>
 struct hash<utils::UUID> {
```